TCMSApp/src/client/app/core/config.js

```diff
@@ -18,6 +18,7 @@
 
         delete window.moment;
         delete window.faker;
+        delete window.Trello;
```

Comment:
`Trello` is something exposed by https://api.trello.com/1/client.js?key=50c269d155e10e2d93a95a16c3b2d9b3 ?